src/Chapter1.hs

```diff
@@ -631,8 +642,22 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
+-- getLast2:: Int -> (Int, Int)
+-- getLast2 y =
+--     let lastN = y `mod` 10
+--         secondLastN = y `mod` 100
+--     in(lastN, secondLastN)
+
+
+sumLast2:: Int -> Int
+sumLast2 n  
+    | n == 0 = 0
+    | n < 10 = n
```

Comment:
Nice implementation! But this guard will return the negative number itself on the negative input